cdc_fuse_fs/main.cc

```diff
@@ -136,9 +161,16 @@ int main(int argc, char* argv[]) {
   printf("%s\n", cdc_ft::kFuseUpToDate);
   fflush(stdout);
 
+  // Create mount dir if it doesn't exist yet.
+  absl::Status status = cdc_ft::CreateMountDir(mount_args);
```

Comment:
Frankly, I don't think we should be creating the mount directory. I've never seen any Linux command to create the target mount directory for you. The mount directory always must exist, otherwise it's a user error. We should follow those conventions.

Reply:
I agree that if you think of cdc_stream as a mount tool, it's uncommon to create dirs. However, that's not how I see it. I think it's more similar to cdc_rsync, just that it doesn't copy, but instead it streams the data. For instance
  cdc_rsync -r C:\path\to\dir host:~/target_dir
creates target_dir. Likewise, I think it makes sense if
  cdc_stream start C:\path\to\dir host:~/target_dir
also creates target_dir. IMHO, the fact that it uses mounts is an implementation detail. WDYT?

Reply:
I still think creating the mount directory is not a good idea. When you sync the data, it is persisted on the target, so all files and dirs need to be created. When you stream a directory, nothing is persisted (cache aside), but the empty mount directory remains as an artifact.

To stay in that logic, if we created the mount directory, we should delete it again at the end of the session. To be clear, I'm not recommending that we do that, I suggest we neither create nor delete the target directory for streaming.

Reply:
We consulted a third person offline to tie break and decided to keep creating the dir.